### PR TITLE
Update EIP-223: Move to Review

### PR DESCRIPTION
**Pull request by Dexaran**
When opening a pull request to submit a new EIP, please use the suggested template: https://github.com/ethereum/EIPs/blob/master/eip-template.md

We have a GitHub bot that automatically merges some PRs. It will merge yours immediately if certain criteria are met:

 - The PR edits only existing draft PRs.
 - The build passes.
 - Your GitHub username or email address is listed in the 'author' header of all affected PRs, inside <triangular brackets>.
 - If matching on email address, the email address is the one publicly listed on your GitHub profile.

### CODE REVIEW

**Review by Pandapip1 — CHANGES_REQUESTED** [review]
The title cannot contain the EIP number

File: EIPS/eip-223.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ title: 223 Token with communication model
 description: Token with event handling and communication model
```
Comment (xinbenlv) [inline EIPS/eip-223.md:4]:
```
- 223 Token with communication model
+ Token with communication model
```

Comment (Dexaran) [reply EIPS/eip-223.md:4]: Resolved https://github.com/ethereum/EIPs/commit/05c18b729fbac4af0b6efb950b3dc7cfdbaaa8f0

**Review by Pandapip1 — APPROVED** [review]
Fine for now.

**Comment by eip-automerger** [comment]
Hi! I'm a bot, and I wanted to automerge your PR, but couldn't because of the following issue(s):

 - Trying to change EIP 223 state from Draft to Review

**Pull request merged**